views/index.ejs

```diff
@@ -60,10 +60,11 @@
 <div id="known-caveats" class="paper">
   <p>Known caveats:</p>
   <ul>
-    <li>Detection for features under prefixes are not yet supported</li>
+    <li>Detection for features under prefixes are not fully supported</li>
     <li>Detection for features under alternative names are not yet supported</li>
     <li>In older browsers, some attributes aren't initialized in the API prototypes, resulting in false negatives (see <a href="https://github.com/foolip/mdn-bcd-collector/issues/406">#406</a>)</li>
     <li>Some APIs are accessed through other APIs and aren't exposed on their own; custom tests have not been written for them all</li>
+    <li>The test page uses a JSON polyfill for core functionality; tests for <code>javascript.builtins.JSON</code> should not be trusted</li>
```

Comment:
This didn't change in this PR, but seems serious. Is there any issue filed about this? We should delete the tests if they can't be trusted.